modules/express/test/unit/typedRoutes/decode.ts

```diff
@@ -190,4 +194,24 @@ describe('io-ts decode tests', function () {
     // valid body
     assertDecode(t.type(UnlockLightningWalletBody), { passphrase: 'secret' });
   });
+
+  it('express.keychain.changePassword', function () {
```

Comment:
Maybe we should start moving each api decode test to their own file, you'd get less merge conflicts that way, and it would organize the tests better.